components/kyma-environment-broker/internal/event-hub/event_hub_step.go

```diff
@@ -0,0 +1,209 @@
+package event_hub
```

Comment:
all provisioning step should be inside `/internal/process/provisioning/` dir

Reply:
Done. Furthermore, I moved the azure package to `/internal/hyperscaler`. Let me know if you have a different suggestion :)